src/pages/get-eth.js

```diff
@@ -25,18 +25,16 @@ import {
 } from "../components/SharedStyledComponents"
 
 const Subtitle = styled.div`
-  font-size: 20px;
+  font-size: 1.25rem;
   line-height: 140%;
+  max-width: 45ch;
   text-align: center;
   color: ${(props) => props.theme.colors.text200};
-`
 
-const SubtitleTwo = styled.div`
-  font-size: 20px;
-  line-height: 140%;
-  margin-bottom: 2rem;
-  text-align: center;
-  color: ${(props) => props.theme.colors.text300};
+  /* Add margin-bottom to the second subtitle */
+  &:nth-last-of-type(2) {
+    margin-bottom: 2rem;
+  }
```

Comment:
Do we want this on the "second subtitle" (`&:nth-of-type(2)`), or do we want this on the _last_ subtitle (`&:last-of-type`)? Not sure we want to use `&:nth-last-of-type(2)`..

If we just want the margin to be on the last item:
```suggestion
  /* Add margin-bottom to the second subtitle */
  &:last-of-type {
    margin-bottom: 2rem;
  }
```